### DISCUSSION TITLE
Remove duplicates when merging different css

### DISCUSSION THREAD

**Discussion by kingcw** [unocss/unocss#1343] [Q&A] [answered]
Hi,

Is there a way to remove duplicates when merging different css?  For example

library A with dist css A

in library B index.ts
```js
import 'A/A.css'
```

after building library B, css B now includes css A, but there are duplicates

Is there a way to remove duplicates?

Thank you.

**Comment by antfu** [comment, marked as answer]
Sorry how is this related to UnoCSS?

**Reply by kingcw** [reply]
yeah it should be a vite plugin.

Thank you.
